atlasdb-client/src/main/java/com/palantir/atlasdb/cache/TimestampCache.java

```diff
@@ -15,26 +15,34 @@
  */
 package com.palantir.atlasdb.cache;
 
+import java.util.concurrent.ConcurrentMap;
+
 import javax.annotation.Nullable;
 
 import com.codahale.metrics.MetricRegistry;
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.cache.Cache;
 import com.google.common.cache.CacheBuilder;
+import com.palantir.atlasdb.AtlasDbConstants;
 import com.palantir.atlasdb.util.AtlasDbMetrics;
 
 /**
  * This class just here for readability and not directly leaking / tying us down to a Guava class in our API.
  */
 public class TimestampCache {
 
-    private final Cache<Long, Long> startToCommitTimestampCache;
+    private Cache<Long, Long> startToCommitTimestampCache;
 
-    public static TimestampCache create() {
```

Comment:
This is a (admittedly minimal) devbreak, so should probably mention it in the release notes.